docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db && python manage.py migrate && python manage.py collectstatic --noinput && python manage.py runserver 0.0.0.0:8000"
+    env_file:
+        - .env
+    volumes:
+      - static_volume:/app/static
+      - media_volume:/app/media
+      - staticfiles:/app/staticfiles
```

Comment:
The use of both `static_volume` and `staticfiles` for the same purpose might be redundant. Ensure that these volumes are necessary and correctly configured.